type-tests/api.ts

```diff
@@ -67,6 +67,13 @@ import {
   DeprecationFailure,
   Warning,
   Target,
+  // Helper hooks
+  registerHook,
+  runHooks,
+  Hook,
+  HookHelperName,
+  HookLabel,
+  HookUnregister,
```

Comment:
Now here we can again drop the unneeded exports; we should also `import type` as appropriate:

```suggestion
  type Hook,
  type HookUnregister,
```

Reply:
As an aside, we should consider adding this lint rule to the project to catch these: https://typescript-eslint.io/rules/consistent-type-imports/